Lucy

heavy-lifting opened the pull request:
hope this is okay! Git confuses me even though I use it every day :/

TristanCacqueray commented:
Welcome!

TodePond commented:
it's okay :) i added you as collaborator so feel free to edit/add files directly if you'd like

heavy-lifting commented:
Cheers I will :)

On Tue, 25 Feb 2025, 00:12 Lu Wilson, ***@***.***> wrote:

> it's okay :) i added you as collaborator so feel free to edit/add files
> directly if you'd like
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pastagang/pastagang/pull/20#issuecomment-2679998597>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE3ULXODMSHYNMNFVKSTGVT2ROYPBAVCNFSM6AAAAABXZIBXNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNZZHE4TQNJZG4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: TodePond]*TodePond* left a comment (pastagang/pastagang#20)
> <https://github.com/pastagang/pastagang/pull/20#issuecomment-2679998597>
>
> it's okay :) i added you as collaborator so feel free to edit/add files
> directly if you'd like
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pastagang/pastagang/pull/20#issuecomment-2679998597>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE3ULXODMSHYNMNFVKSTGVT2ROYPBAVCNFSM6AAAAABXZIBXNSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNZZHE4TQNJZG4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
